api/client/gro_client.py

```diff
@@ -1082,9 +1111,16 @@ def GDH(self, gdh_selection, **optional_selections):
         entity_ids = [int(x) for x in gdh_selection.split("-")]
         selection = zip_selections(entity_ids, **optional_selections)
 
-        self.add_single_data_series(selection)
+        gdh_series = self.get_data_series(**selection)
```

Comment:
we don't want to call get_data series here, as this is at the data retrieval "layer"  while get_data_series is exploration layer.  also don't want to change the semantics of this function... it will return a single series because gdh string is specified that way and this is a selection.

all we would need here is to convert `selection` into `entity names` i.e. get a list of (entity_type, entity_id) from selections and  `entity_names = [self.lookup(entity_type, entity_id) ....]`

Reply:
that said, see comment in get_df(),  entity names are not unique in general so we can't really ise them as the index to get series, we need ids and then have names as just an option output.  so this function can remain unchanged.

Reply:
reverted the GDH changes